dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarClassPathEntry.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import java.nio.file.Path;
+import java.util.Objects;
+
+/** JAR file entry in a class path. */
+class JarClassPathEntry implements ClassPathEntry {
+  private Path jar;
+
+  JarClassPathEntry(Path jar) {
+    this.jar = jar;
+  }
+
+  @Override
+  public String getClassPath() {
```

Comment:
This is a path as a string, not a class path. A path looks like "/usr/lib/classes/foo.jar"
A classpath is a semicolon or colon separated list of paths.

Reply:
A class path consists of smaller class paths.

```
-classpath classpath1;classpath2...
```
https://docs.oracle.com/javase/7/docs/technotes/tools/windows/classpath.html

That's why I named `getClassPath`. Do you think of better name?

Reply:
A classpath consists of class paths entries, not classpaths. And a classpath entry is a path.

Reply:
Then `String getPath()`.